cl/opinion_page/templates/view_opinion_summaries.html

```diff
@@ -0,0 +1,54 @@
+{% extends "base.html" %}
+{% load text_filters %}
+{% load humanize %}
+
+{# Shown on authorities pages. #}
+
+{% block title %}Summaries of {{ title }} – CourtListener.com{% endblock %}
+{% block og_title %}Summaries of {{ title }} –
+    CourtListener.com{% endblock %}
+{% block description %}Summaries of {{ title }}{% endblock %}
+{% block og_description %}Summaries of {{ title }}{% endblock %}
+
```

Comment:
We could do a little better here, no? This og_description is what will show up in Slack when you share this link. Maybe say something like, "22 summaries of case {{title}} were extracted from other cases. The top one says that, "xyz"". That'd be neat, right?